packages/gatsby-cli/src/reporter/reporter.ts

```diff
@@ -256,5 +256,5 @@ class Reporter {
   // "reporter._setStage is not a function" error when gatsby@<2.16 is used with gatsby-cli@>=2.8
   _setStage = (): void => {}
 }
-
+export type { Reporter }
```

Comment:
I'm doing `export type` because we don't want to give access to the class itself, just the type.

Reply:
smart, i like this